app/scripts/lib/setupSentry.js

```diff
@@ -86,7 +86,12 @@ function getClientOptions() {
     integrations: [
       Sentry.dedupeIntegration(),
       Sentry.extraErrorDataIntegration(),
-      Sentry.browserTracingIntegration(),
+      Sentry.browserTracingIntegration({
+        shouldCreateSpanForRequest: (url) => {
+          // Do not create spans for outgoing requests to a 'sentry.io' domain.
+          return !url.match(/sentry\.io\/?$/u);
+        },
+      }),
```

Comment:
For comparison, this is the code example from the docs:

> `shouldCreateSpanForRequest`
>
> This function can be used to filter out unwanted spans such as XHRs running health checks or something similar. If this function isn't specified, spans will be created for all requests.
>
>```ts
>    Sentry.browserTracingIntegration({
>      shouldCreateSpanForRequest: (url) => {
>        // Do not create spans for outgoing requests to a `/health/` endpoint
>        return !url.match(/\/health\/?$/);
>      },
>    }),
>```
> — https://docs.sentry.io/platforms/javascript/tracing/instrumentation/automatic-instrumentation/#shouldcreatespanforrequest

Reply:
This regex is incorrect.  Because it has the `$`, it will only match URLs that end in `sentry.io/`

Reply:
You're right! This could be fixed by making the regex more specific:

```suggestion
      Sentry.browserTracingIntegration({
        shouldCreateSpanForRequest: (url) => {
          // Do not create spans for outgoing requests to a 'sentry.io' domain.
          return !url.match(
            /^https?:\/\/(\w+@)?(\w+\.ingest\.\w{2,3}\.)?sentry\.io(\/api)?/u,
          );
        },
      }),
```

But I guess a simpler, more permissive regex would be better performance-wise (less backtracking and potentially exponential-time greedy matching). 

```suggestion
      Sentry.browserTracingIntegration({
        shouldCreateSpanForRequest: (url) => {
          // Do not create spans for outgoing requests to a 'sentry.io' domain.
          return !url.match(/sentry\.io/u);
        },
      }),
```

What do you think? The only possible problem I can think of from taking the second approach would be false positives being filtered out, but that's probably not an issue with the Sentry domain.

Reply:
That's a good point. Maybe we could strike a middle ground by just checking that we're in the right domain? 

```suggestion
      Sentry.browserTracingIntegration({
        shouldCreateSpanForRequest: (url) => {
          // Do not create spans for outgoing requests to a 'sentry.io' domain.
          return !url.match(
            /^https?:\/\/([\w\d.@-]+\.)?sentry\.io(\/|$)/u
          );
        },
      }),
```

(Not sure if the $ in capture group at the end is correct syntax)

Or we could just go more specific, since the security concern seems like a higher priority than performance gains here.